### ISSUE TITLE
Problem Running protonvpn

### ISSUE THREAD

**Issue by Timmy-Tenders** [ProtonVPN/linux-cli#57]
When trying to run any command even ```protonvpn-cli --version``` the following error is thrown:

Traceback (most recent call last):
  File "/usr/bin/protonvpn-cli", line 33, in <module>
    sys.exit(load_entry_point('protonvpn-cli==3.11.0', 'console_scripts', 'protonvpn-cli')())
  File "/usr/lib/python3.10/site-packages/protonvpn_cli/main.py", line 20, in main
    from .cli import ProtonVPNCLI
  File "/usr/lib/python3.10/site-packages/protonvpn_cli/cli.py", line 4, in <module>
    from proton.constants import VERSION as proton_version
ModuleNotFoundError: No module named 'proton'

On Arch Linux Python3 Version 3.10.1

**Comment by ciaranbor** [comment]
This happened to me today too. 

Solved it by uninstalling all pvpn packages, removing any cached stuff (probably only `~/.cache/protonvpn` matters but I removed `yay` caching too), and reinstalling.

**Comment by Timmy-Tenders** [comment]
Which packages did you remove ? I did reinstall removing the yay cache as well as yay cache.

**Comment by ciaranbor** [comment]
The AUR metapackage (`protonvpn`). It is a little funny, I had to do the process twice on one of my systems.

**Comment by Timmy-Tenders** [comment]
That last one worked for me thanks.

**Comment by eljejer** [comment]
The last one worked for me too. Thanks,

**Comment by zerotobtc** [comment]
For me too. I think this can be closed!

**Comment by abinlatheef** [comment]
Didn't work for me. This is the second time. Last time it worked when I reinstalled.

**Comment by Sn0wAlice** [comment]
Hey !
i'have the same problem. BUT any of your solution or solution of [this link](https://www.reddit.com/r/ProtonVPN/comments/rknt1s/protonvpn_not_working_on_arch_linux/) works for me. 

Exactly got this error: 
```
Traceback (most recent call last):
  File "/usr/bin/protonvpn-cli", line 33, in <module>
    sys.exit(load_entry_point('protonvpn-cli==3.11.0', 'console_scripts', 'protonvpn-cli')())
  File "/usr/bin/protonvpn-cli", line 25, in importlib_load_entry_point
    return next(matches).load()
StopIteration
```

**Comment by Delan168** [comment]
Hello!
I propose to re-open this issue since exactly the same still happens on Manjaro (fully updated).

Tried the following:
- `pamac remove -o protonvpn-cli`
- Cleaning of pamac cache (done through GUI: preference, third parties, ...)
- `rm -rf ~/.cache/protonvpn/`
- `pamac build protonvpn-cli`

Builds without error, however any `protonvpn-cli` commande returns
```
Traceback (most recent call last):
  File "/usr/bin/protonvpn-cli", line 33, in <module>
    sys.exit(load_entry_point('protonvpn-cli==3.11.0', 'console_scripts', 'protonvpn-cli')())
  File "/usr/lib/python3.10/site-packages/protonvpn_cli/main.py", line 20, in main
    from .cli import ProtonVPNCLI
  File "/usr/lib/python3.10/site-packages/protonvpn_cli/cli.py", line 4, in <module>
    from proton.constants import VERSION as proton_version
ModuleNotFoundError: No module named 'proton'
```
Linux 5.10.89-1-MANJARO with Python 3.10.1
Thread on Manjaro Forum: [Proton vpn stopped working](https://forum.manjaro.org/t/proton-vpn-stopped-working/95569/24)

Any clue on how to solve this? What would be the "pamac equivalent" of `yay -Rs`?

**Comment by ParaplegicRacehorse** [comment]
... so `pamac build #(pacman -Qoq /usr/lib/python3.9)` returns:   (terminal output attached as txt file)

[protonvpn-pythonup-pamacerrors.txt](https://github.com/ProtonVPN/linux-cli/files/8149319/protonvpn-pythonup-pamacerrors.txt)

tl;dr:
build tests fail.  install aborts.

**Comment by ParaplegicRacehorse** [comment]
Update.

Further testing revealed, in my case, the AUR python package build string failed due to other packages.  Attempting `pamac build` and giving only the python-proton packages as argument worked.

See here: https://forum.manjaro.org/t/stable-update-2022-02-27-kernels-mesa-21-3-7-plasma-5-24-2-frameworks-5-91-pipewire-0-3-47-toolchain-gstreamer-1-20-nvidia-510-54/103923/69?u=paraplegicracehorse

**Comment by tuxedotje** [comment]
did solve this by install the protonvpn-beta.

**Comment by vaibhav-mzp** [comment]
I'm getting this error:

`Traceback (most recent call last):
  File "/usr/bin/protonvpn", line 11, in <module>
    load_entry_point('protonvpn-gui==1.8.0', 'console_scripts', 'protonvpn')()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 479, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2861, in load_entry_point
    return ep.load()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2465, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2471, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/lib/python3/dist-packages/protonvpn_gui/main.py", line 14, in <module>
    from proton.constants import VERSION as proton_version
  File "/usr/lib/python3/dist-packages/proton/__init__.py", line 1, in <module>
    from .api import Session # noqa
  File "/usr/lib/python3/dist-packages/proton/api.py", line 6, in <module>
    import requests
  File "/usr/lib/python3/dist-packages/requests/__init__.py", line 43, in <module>
    import urllib3
  File "/usr/local/lib/python3.10/dist-packages/urllib3/__init__.py", line 8, in <module>
    from .connectionpool import (
  File "/usr/local/lib/python3.10/dist-packages/urllib3/connectionpool.py", line 29, in <module>
    from .connection import (
  File "/usr/local/lib/python3.10/dist-packages/urllib3/connection.py", line 39, in <module>
    from .util.ssl_ import (
  File "/usr/local/lib/python3.10/dist-packages/urllib3/util/__init__.py", line 3, in <module>
    from .connection import is_connection_dropped
  File "/usr/local/lib/python3.10/dist-packages/urllib3/util/connection.py", line 3, in <module>
    from .wait import wait_for_read
  File "/usr/local/lib/python3.10/dist-packages/urllib3/util/wait.py", line 1, in <module>
    from .selectors import (
  File "/usr/local/lib/python3.10/dist-packages/urllib3/util/selectors.py", line 14, in <module>
    from collections import namedtuple, Mapping
ImportError: cannot import name 'Mapping' from 'collections' (/usr/lib/python3.10/collections/__init__.py)
`

**Comment by vaibhav-mzp** [comment]
Just solved the problem by deleting "/usr/local/lib/python3.10" folder and installing required modules manually by pip in a non-root terminal.

**Comment by cyberksh** [comment]
I got it working by doing the following:
```
pamac remove python-proton-client python-protonvpn-nm-lib protonvpn-gui protonvpn-cli protonvpn
pamac clean -k 0 # removes build files that are cached
rm -rf ~/.cache/protonvpn
pamac install protonvpn
```
